docs/modules/ROOT/pages/java.adoc

```diff
@@ -857,13 +830,16 @@ explicitly enable them in the load balancer's configuration (see
 https://developer.couchbase.com/documentation/mobile/1.5/guides/sync-gateway/nginx/index.html[Load
 Balancers]).
```

Comment:
This needs to be updated to or this is as expected?
https://developer.couchbase.com/documentation/mobile/2.0/installation/sync-gateway/index.html